### ISSUE TITLE
Standard TTS generation example from the documentation does not work

### ISSUE THREAD

**Issue by waan1** [erew123/alltalk_tts#293]
**Describe the bug**
Narrator Example example works well, but Standard TTS generation example fails.

**To Reproduce**
The configuration is exactly the same as in my previous issue report - problem with Polish language #292

curl -X POST "http://127.0.0.1:7851/api/tts-generate" -d "text_input=All of this is text spoken by the character. This is text not inside quotes, though that doesnt matter in the slightest" -d "text_filtering=standard" -d "character_voice_gen=female_01.wav" -d "narrator_enabled=false" -d "narrator_voice_gen=male_01.wav" -d "text_not_inside=character" -d "language=en" -d "output_file_name=myoutputfile" -d "output_file_timestamp=true" -d "autoplay=true" -d "autoplay_volume=0.8"
{"status":"generate-failure","error":"An error occurred"}%

[AllTalk TTSGen] 2.58 seconds. LowVRAM: False DeepSpeed: False
[AllTalk TTSGen] All of this is text spoken by the character. This is text not inside quotes, though that doesnt matter in the slightest

**Screenshots**

**Text/logs**
If applicable, copy/paste in your logs here from the console.

**Desktop (please complete the following information):**
AllTalk was updated: [approx. date]
Custom Python environment: [yes/no give details if yes]
Text-generation-webUI was updated: [approx. date]

**Additional context**

**Comment by waan1** [comment]
I see that it fails when narrator_enabled is not true
Also if autoplay_volume is not defined even if autoplay=false

**Comment by erew123** [comment]
Hi @waan1 

Not sure I can re-create the fault you're describing. I have copied/pasted the exact CURL command from above, I have also altered the narrator_enabled and the autoplay:

![image](https://github.com/user-attachments/assets/a4a1af0e-b805-4638-866d-c2de18553ddc)

All works fine.

Ive had no other complaints or issues like this before, so Im a little lost as to what to suggest could be the exact problem. What I could suggest is moving to the V2 beta as there is extra levels of debugging available in that one, plus there are very beneficial changes to the API that simplify generation.

https://github.com/erew123/alltalk_tts/tree/alltalkbeta
